package-lock.json

```diff
@@ -1997,7 +2002,8 @@
     "follow-redirects": {
       "version": "1.15.4",
       "resolved": "https://registry.npmjs.org/follow-redirects/-/follow-redirects-1.15.4.tgz",
-      "integrity": "sha512-Cr4D/5wlrb0z9dgERpUL3LrmPKVDsETIJhaCMeDfuFYcqa5bldGV6wBsAN6X/vxlXQtFBMrXdXxdL8CbDTGniw=="
+      "integrity": "sha512-Cr4D/5wlrb0z9dgERpUL3LrmPKVDsETIJhaCMeDfuFYcqa5bldGV6wBsAN6X/vxlXQtFBMrXdXxdL8CbDTGniw==",
+      "dev": true
     },
```

Comment:
Shouldn't this dependency also be updated?
I think, the version specified here is used for jest-puppeteer:
```
$ npm ls follow-redirects
aws-crt@1.0.0-dev aws-crt-nodejs
├─┬ axios@1.6.8
│ └── follow-redirects@1.15.6 
└─┬ jest-puppeteer@5.0.4
  └─┬ jest-environment-puppeteer@5.0.4
    └─┬ jest-dev-server@5.0.3
      └─┬ wait-on@5.3.0
        └─┬ axios@0.21.4
          └── follow-redirects@1.15.4          <== old version
```

Reply:
Dev dependencies aren't a concern here.